test/com/arpnetworking/metrics/portal/TestUtil.java

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright 2016 Groupon.com
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.arpnetworking.metrics.portal;
+
+import com.google.common.base.Optional;
+import models.internal.Alert;
+import models.internal.Expression;
+import models.internal.NagiosExtension;
+import org.joda.time.Period;
+
+import java.util.Objects;
+import java.util.concurrent.atomic.AtomicInteger;
+
+/**
+ * Utility class for tests.
+ *
+ * @author Deepika Misra (deepika at groupon dot com)
+ */
+public final class TestUtil {
+
+    /**
+     * Generates the JDBC url with a unique port number and database name.
+     *
+     * @return The JDBC url.
+     */
+    public static String generateJdbcUrl() {
+        final int nextCounter = UNIQUE_COUNTER.getAndIncrement();
+        final int port = BASE_PORT + nextCounter;
+        return "jdbc:h2:./target/h2/metrics" +
+                nextCounter +
+                ":portal;AUTO_SERVER=TRUE;AUTO_SERVER_PORT=" +
+                port +
+                ";MODE=PostgreSQL;INIT=create schema if not exists portal;DB_CLOSE_DELAY=-1";
+    }
+
+    public static boolean isAlertEbeanEquivalent(final Alert alert, final models.ebean.Alert ebeanAlert) {
```

Comment:
This is only used in one test file. Why does it need to be in a utility class? (remember: 1-2-refactor)